include/sddf/benchmark/sel4bench.h

```diff
@@ -196,6 +196,11 @@ static FASTFN seL4_Word sel4bench_get_num_counters()
 
 static FASTFN void sel4bench_init()
 {
+    // do kernel-mode PMC init
+// #ifndef CONFIG_EXPORT_PMU_USER
```

Comment:
What happened with this?

I understand the `ifndef` looks wrong but I can't tell why this change is suddenly needed compared to before.

Reply:
This is just a copy of the fix which we used to allow the PMU data to be correct in hypervisor mode. However we did not realise that the system could no longer build when not in benchmark mode since `sel4bench_private_init` is not defined. After looking into this further, we should not need to run this function in our system anyway, so I will remove these commented lines of code.